### PR TITLE
TSK-1657: Angular 12 Update

### PR DESCRIPTION
**Pull request by Tristan2357**
Updated Angular to 12
Also updated most dependencies

---
 
### For the submitter:
- [ ] I updated the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) and will supply links to the specific files
- [x] I did not update the [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview)
- [ ] I included a link to the [SonarCloud branch analysis](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1019969636/SonarCloud+Integration)
- [x] I added a description of changes on the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [ ] I did not update the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana)
- [x] I put the ticket in review
- [ ] After integration of the pull request, I verified our [bluemix test environment](http://taskana.mybluemix.net/taskana) is not broken

### Verified by the reviewer:
- [ ] Commit message format → TSK-XXX: Your commit message.
- [ ] Submitter's update to [documentation](https://taskana.atlassian.net/wiki/spaces/TAS/overview) is sufficient
- [ ] SonarCloud analysis meets our standards
- [ ] Update of the [current release notes](https://taskana.atlassian.net/wiki/spaces/TAS/pages/1281392672/Current+Release+Notes+Taskana) reflects changes
- [ ] PR fulfills the ticket
- [ ] Edge cases and unwanted side effects are tested
- [ ] Readability

### CODE REVIEW

File: web/package.json [hunk 1]
```diff
@@ -20,57 +21,54 @@
...
+    "@angular/core": "12.0.5",
+    "@angular/forms": "12.0.5",
+    "@angular/material": "12.0.5",
+    "@angular/platform-browser": "12.0.5",
+    "@angular/platform-browser-dynamic": "12.0.5",
+    "@angular/router": "12.0.5",
+    "@circlon/angular-tree-component": "11.0.4",
+    "@ngxs/store": "3.7.2",
+    "angular-svg-icon": "12.0.0",
     "chart.js": "2.9.4",
```
Comment (mustaphazorgati) [inline web/package.json:37]: version 3.3.2 exists: https://www.npmjs.com/package/chart.js

**Review by mustaphazorgati — CHANGES_REQUESTED** [review]

File: web/package.json (hunk 1 above)
Comment (cnguyen-de) [reply web/package.json:37]: I think chart.js isn't installed directly but as dependency of ng2-charts which has not as up-to-date version

**Review by cnguyen-de — APPROVED** [review]

File: web/package.json (hunk 1 above)
Comment (mustaphazorgati) [reply web/package.json:37]: Then it is a transitive dependency? -> remove?

Comment (mustaphazorgati) [reply web/package.json:37]: @cnguyen-de you are right: https://github.com/valor-software/ng2-charts/blob/2.4.2/package.json

Comment (Tristan2357) [reply web/package.json:37]: The install warnings indicate, that chart.js is a required peer dependency, so it needs to stay

Comment (mustaphazorgati) [reply web/package.json:37]: should we comment that? :)

**Review by mustaphazorgati — APPROVED** [review]

**Pull request merged**